daemons/gptp/common/ptptypes.hpp

```diff
@@ -36,7 +36,12 @@
 
 /**@file*/
 
+#if defined(__clang__) &&  defined(__x86_64__)
+// Clang/llvm has incompatible long double (fp128) for x86_64.
+typedef double FrequencyRatio; /*!< Frequency Ratio */
+#else
```

Comment:
This should be a separate commit? It seems un-related to getting 32-bit to compile and run.